web/pdf_page_view.js

```diff
@@ -468,19 +472,15 @@ class PDFPageView {
     if (!this.textLayer) {
```

Comment:
Given the changes below, please let's ensure that the structTreeLayer is available when this method is invoked.
```suggestion
    if (!this.textLayer || !this.structTreeLayer) {
```